src/components/Card/index.jsx

```diff
@@ -7,14 +7,21 @@ class Card extends Component {
         isChecked: false,
         isEdit: false,
         cardData : {
-            Logo: 'Caption',
-            Text: 'Some text ...',
+            Logo: this.props.head,
+            Text: this.props.body,
         },
         tempCardData: {
             
         }
         
     }
+    static getDerivedStateFromProps(nextProps, prevState){
+        if (nextProps.viewMode === true) {
+        return {isEdit: false}
+        }
+        return prevState
+        }
+    
```

Comment:
Why not use ES6 arrow functions, boolean values compact comparison, ternary operator? Would be much shorter and easier to operate with:

`static getDerivedStateFromProps = (nextProps, prevState) => nextProps.viewMode ? {isEdit: false} : prevState;`